build.go

```diff
@@ -47,6 +47,11 @@ func Build(entryResolver EntryResolver, dependencyManager DependencyManager, sbo
 		}
 
 		entry, allEntries := entryResolver.Resolve("node", context.Plan.Entries, priorities)
+		if entry.Name == "" && len(allEntries) == 0 {
+			logger.Process("Node no longer requested by plan")
+			return packit.BuildResult{}, nil
+		}
+		logger.Process("Resolving Node Engine version")
```

Comment:
The integration tests are failing because this log line is output twice. It already appears on line 39. I think we should keep the original line and remove this new copy.

Reply:
Thanks, agreed.

Reply:
@ryanmoran udpated.

Reply:
@ryanmoran just to confirm I should be able to just run the integration tests locally by running the script?  If I can I'd like to do that for any future PRs. If that's the case I'll try to figure out why they did not run locally for me.

Reply:
@mhdawson yes. I'd be happy to help you debug `./scripts/integration.sh` if that isn't working for you.

Reply:
I setup a new environment and they seem to run for me, not sure what was missing before, but thanks for confirming they should have just worked.